### PR TITLE
README.md: mention caskbot

### PR DESCRIPTION
**Pull request by vitorgalvao**
See https://github.com/caskroom/caskbot/issues/4 for context.

### CODE REVIEW

**Comment by adityadalal924** [comment]
Should we also add a link to the gitter chat here? There's a badge for it, but I don't know if people really use it

**Comment by vitorgalvao** [comment]
> Should we also add a link to the gitter chat here? There's a badge for it, but I don't know if people really use it

I don’t see why not. It seems to make sense for the section.

**Comment by adityadalal924** [comment]
Now a merge conflict, but otherwise LGTM

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ We’re really rather friendly! Here are the best places to talk about the proje
   * [Bug report][bug_report_template]
   * [Feature request][feature_request_template]
   * [Cask request][cask_request_template]
-* Join us on IRC, we’re at `#homebrew-cask` on Freenode
+* Join us (and [caskbot](https://github.com/passcod/caskbot)) on IRC, we’re at `#homebrew-cask` on Freenode
```
Comment (jawshooah) [inline README.md]: Should either add a period here or remove it on the Gitter line for consistency.

Comment (vitorgalvao) [inline README.md]: Agreed. I added the period by reflex.

**Comment by vitorgalvao** [comment]
Merged as bbc26347fae4abae33d3908670856176cf44cf06.

**Pull request closed without merging**